src/Symfony/Bundle/FrameworkBundle/Command/ServerStartCommand.php

```diff
@@ -129,7 +134,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        if (null === $process = $this->createServerProcess($output, $address, $documentRoot, $input->getOption('router'), $env, null)) {
+        if (null === $process = $this->createServerProcess($output, $address, $documentRoot, $router, null)) {
```

Comment:
Could do, though the `null` would still need passing to the `Process` constructor, otherwise the timeout will be set to 60 seconds:
```php
private function createServerProcess(OutputInterface $output, $address, $documentRoot, $router)
{
    ...
    ...
    return new Process('exec '.$script, $documentRoot, null, null, null);
}
```
It doesn't seem to do any harm leaving the option there for future use, but then, you're right, it's currently not being used to doesn't need to be there.  Let me know if you'd prefer for it to be removed and, if so, I'll take it out.